netutils/config/compliance.py

```diff
@@ -277,8 +278,8 @@ def diff_network_config(compare_config: str, base_config: str, network_os: str)
             for parent in line.parents:
                 if parent not in needed_lines:
                     needed_lines.append(parent)
-            needed_lines.append(line.config_line)
-
+            if line.config_line:
```

Comment:
What use case and associated test is this accounting for?

Reply:
If there are two banners (`motd` & `exec`) in either the intended or backup config https://github.com/scrapli/scrapli_community/tree/main/scrapli_community/aruba/aoscx but only one in the other config https://github.com/networktocode/netutils/pull/172/files#diff-3df9a197865dd3631c2da5ff59f624886dd2bf16ed16d7fe746b25c1d126f566R2-R4 This will throw a TypeError 
```
Traceback (most recent call last):
  File "/netutils/./test.test", line 68, in <module>
    comply = compliance(features, running_config, generated_config, "aruba_aoscx", "string")
  File "/netutils/netutils/config/compliance.py", line 188, in compliance
    compliance_results.update({feature["name"]: feature_compliance(feature, backup_str, intended_str, network_os)})
  File "/netutils/netutils/config/compliance.py", line 338, in feature_compliance
    feature_data.update(_check_configs_differences(intended_cfg, backup_cfg, network_os))
  File "/netutils/netutils/config/compliance.py", line 66, in _check_configs_differences
    extra = diff_network_config(actual_cfg, intended_cfg, network_os)
  File "/netutils/netutils/config/compliance.py", line 283, in diff_network_config
    return "\n".join(needed_lines)
TypeError: sequence item 4: expected str instance, NoneType found
```